pages/dkp/kommander/1.4/projects/project-deployments/index.md

```diff
@@ -35,31 +35,29 @@ This section requires that Dispatch is installed, and, optionally, you can confi
 
 ![Continuous Deployment (CD)](/dkp/kommander/1.4/img/project-cd-welcome-screen.png)
 
-Before we setup our GitOps source, we need to create a secret that can be used to access our GitOps source.
-
 ### Setting up a secret for accessing GitOps source.
 
-Create a secret to be used by Dispatch to read the contents of your GitOps repository by creating SCM webhooks:
+Create a secret that Dispatch uses to read the contents of your GitOps repository by creating SCM webhooks:
 
 ![Secret for GitOps source](/dkp/kommander/1.4/img/project-cd-secret-create.png)
 
-Refer to this guide to learn more about [setting up credentials](/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials#setting-up-github-credentials) for Dispatch. Note that the above screen creates a `types.kubefed.io/v1beta1, Kind=FederatedSecret` and this is not yet supported by Dispatch CLI. Please use the GUI as shown above to create a federated secret OR create a `FederatedSecret` manifest and apply it to the project namespace. Learn more about [FederatedSecrets](../project-secrets/).
+Refer to this guide to learn more about [setting up credentials](/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials#setting-up-github-credentials) for Dispatch. Note that the above screen creates a `types.kubefed.io/v1beta1, Kind=FederatedSecret` and this is not yet supported by Dispatch CLI. Use the GUI, as shown above, to create a federated secret or create a `FederatedSecret` manifest and apply it to the project namespace. Learn more about [FederatedSecrets](../project-secrets/).
 
 ### Creating GitOps Source
 
-After the secret is created, you can also view it under the `Secrets` tab. Configure the GitOps source backed by this secret:
+After the secret is created, you can view it in the `Secrets` tab. Configure the GitOps source accessed by this secret:
 
 ![Create GitOps Source](/dkp/kommander/1.4/img/project-cd-gitops-source-create.png)
 
 While the `username` and `password` are mandatory fields, SCM details default to `github` if unspecified. It takes a few seconds for the GitOps Source to be reconciled and the manifests from the SCM repository at the given path to be federated to attached clusters. After the sync is complete, manifests from GitOps source are created in attached clusters.
 
 After the manifests are federated, there are various commands that can be executed from the CLI to check the federation status.
 
-On the manager cluster, check for your `FederatedKustomization` and `FederatedGitRepository` resources and the `status` field reflects the propagation of manifests:
+On the manager cluster, check for your `FederatedKustomization` and `FederatedGitRepository` resources. The `status` field reflects the propagation of manifests:
```

Comment:
Is 'manager cluster' the right term of art here?  I would have though "management cluster" was the right one. 

```suggestion
On the management cluster, check for your `FederatedKustomization` and `FederatedGitRepository` resources. The `status` field reflects the propagation of manifests:
```